README.md

```diff
@@ -27,114 +31,73 @@ IPFS Lite runs the minimal setup required to get and put data on the IPFS networ
-
 ## Install
 
-> Note: `js-ipfs-lite` includes TypeScript type definitions.
-
 ```
-npm i @textile/ipfs-lite
+npm install dag-service
 ```
 
-### Browser
-
-For now, you'll have to bundle your own browser builds, but default builds are coming soon!
-
 ## Usage
 
-Only import what you need, keeping your bundles small and your load times faster. You can grab a full-featured IPFS Lite peer from the top-level library, or grab specific sub-modules as needed:
-
 ```typescript
-// Grab a fully-loaded Peer
-import { Peer, BlockStore } from '@textile/ipfs-lite'
-import { setupLibP2PHost, MemoryDatastore } = from '@textile/ipfs-lite/dist/setup'
-```
-
-### Typescript
-
-```typescript
-import { Peer, BlockStore } from '@textile/ipfs-lite'
+import { Peer, BlockStore } from "dag-service"
 // Use any interface-datastore compliant store
-import { MemoryDatastore } from 'interface-datastore'
-import Libp2p from 'libp2p'
+import { MemoryDatastore } from "interface-datastore"
+import Libp2p from "libp2p"
 
 const store = new BlockStore(new MemoryDatastore())
 
-;(async function() {
-    // Bring your own libp2p host....
-    const host = new Libp2p({ ...libp2Options })
-    // ...or, use a full-featured default host
-    // const host = await setupLibP2PHost()
-    const lite = new Peer(store, host)
-
-    await lite.start()
-
-    const cid = 'QmWATWQ7fVPP2EFGu71UkfnqhYXDYH566qy47CnJDgvs8u'
-    const data = await lite.getFile(cid)
-    console.log(data.toString())
-    // Hello World
-    await lite.stop()
-}
-```
-
-### Nodejs
+const main = async () => {
+  // Bring your own libp2p host....
+  const host = new Libp2p({ ...libp2Options })
+  const lite = new Peer(store, host)
 
-```javascript
-let { Peer, BlockStore } = require('@textile/ipfs-lite')
-let { setupLibP2PHost } = require('@textile/ipfs-lite/dist/setup')
-let { MemoryDatastore } = require('interface-datastore')
-
-let store = new BlockStore(new MemoryDatastore())
-
-;(async function() {
-  let host = await setupLibP2PHost()
-  let lite = new Peer(store, host)
   await lite.start()
 
-  let cid = 'QmWATWQ7fVPP2EFGu71UkfnqhYXDYH566qy47CnJDgvs8u'
-  let data = await lite.getFile(cid)
+  const cid = "QmWATWQ7fVPP2EFGu71UkfnqhYXDYH566qy47CnJDgvs8u"
+  const data = await lite.getFile(cid)
   console.log(data.toString())
   // Hello World
   await lite.stop()
-})()
+}
 ```
 
 There are also several useful examples included in the tests of this repo, with tools for creating a default `libp2p` host exported by default. We've also thrown in some useful interfaces to use when building on IPFS Lite, as well as the Buffer API for use in the browser.
 
 ## API
 
-See [https://textileio.github.io/js-ipfs-lite](https://textileio.github.io/js-ipfs-lite)
+See [https://ipfs.github.io/js-dag-service/](https://ipfs.github.io/js-dag-service/)
 
 ## Maintainers
 
-[Carson Farmer](https://github.com/carsonfarmer)
+- [Irakli Gozalishvili](https://github.com/gozala/)
+- [Carson Farmer](https://github.com/carsonfarmer)
```

Comment:
@carsonfarmer I assumed you still want to be a maintainer of this library, if that is not the case please let me know and I'll remove it.